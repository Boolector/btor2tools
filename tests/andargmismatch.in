1 zero 1
2 ones 2
3 and 3 1 2
