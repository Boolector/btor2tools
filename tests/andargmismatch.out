*** catbtor: parse error in 'andargmismatch.in' line 3: argument lengths do not match
